webapp/channels/src/components/widgets/menu/menu_header.tsx

```diff
@@ -14,17 +14,12 @@ type Props = {
 /**
  * @deprecated Use the "webapp/channels/src/components/menu" instead.
  */
-export default class MenuHeader extends React.PureComponent<Props> {
-    public render() {
-        const {children, onClick} = this.props;
+const MenuHeader: React.FC<Props> = ({ children, onClick }) => {
+    return (
+        <li className='MenuHeader' onClick={onClick}>
+            {children}
+        </li>
+    );
+};
 
-        return (
-            <li
-                className='MenuHeader'
-                onClick={onClick}
-            >
-                {children}
-            </li>
-        );
-    }
-}
+export default MenuHeader;
```

Comment:
It was a pure component so it is expected to have here a `React.memo` to make it memoized.
```suggestion
export default React.memo(MenuHeader);
```